application/app/views/projects/show/_upload_files.html.erb

```diff
@@ -28,13 +28,15 @@
         <%= render partial: '/shared/file_row_date', locals: { date: file.end_date, title: t('.completion_date'), classes: 'mx-2'} if file.end_date %>
         <%= render partial: '/shared/file_row_date', locals: { date: file.start_date, title: t('.download_start_date'), classes: 'mx-2'} if file.status.downloading? %>
 
-        <%= render partial: "shared/button_to", locals: {
+        <%= render layout: "shared/button_to", locals: {
           url: project_upload_bundle_upload_file_path(project_id: bundle.project_id, upload_bundle_id: bundle.id, id: file.id),
           method: 'DELETE',
           title: t(".delete_file"),
           class: 'btn-sm btn-outline-secondary icon-hover-danger',
           icon: "bi bi-trash"
-        } %>
+        } do %>
+          <%= hidden_field_tag :anchor, "tab-#{batch.id}" %>
+        <% end %>
```

Comment:
If the only reason to render it as a layout is to pass the hidden_field_tag, maybe you can pass the hidden field tag as a local variable and render it conditionally in the partial, to keep it using it as a partial. Using this as a Layout it is a bit confusing, because the purpose of layouts is to envelop full views and not to use it as a button component